packages/editor/README.md

```diff
@@ -370,14 +370,328 @@ function Toolbar() {
+```
+
+Next, spread the `coreBehaviors` into the optional `behaviors` array on the `initialConfig` of `EditorProvider`:
+
+```tsx
+<EditorProvider
+  initialConfig={{
+    schemaDefinition,
+    behaviors: [...coreBehaviors],
+  }}
+>
+  {/* ... */}
+</EditorProvider>
+```
+
+Everything should work as before. The only difference is that we've manually defined the core behaviors of the editor.
+
+Now, import `defineBehavior` from `@portabletext/editor/behaviors` and define a new behavior below the core behaviors:
+
+```ts
+import {coreBehaviors, defineBehavior} from '@portabletext/editor/behaviors'
+```
+
+```tsx
+<EditorProvider
+  initialConfig={{
+    schemaDefinition,
+    behaviors: [
+      ...coreBehaviors,
+      defineBehavior({
+        // your behavior
+      }),
+    ],
+  }}
+>
+  {/* ... */}
+</EditorProvider>
+```
+
+This allows you to customize the editor behavior while still allowing the core behaviors to take precedence.
+
+### Overwriting Behaviors
+
+Many of the core behaviors of the editor have already been defined using `defineBehavior` and we'll continue to port core behaviors to this API. This means that if you leave `coreBehaviors` out of the `behaviors` array, then you already get a much more bare-bones editor experience that you can customize as you please. And if you want to pick and choose core behaviors, then you can import the `coreBehavior` object and pick just the core behaviors you want:
+
+```ts
+import {coreBehavior} from '@portabletext/editor/behaviors'
+```
+
+```tsx
+<EditorProvider
+  initialConfig={{
+    schemaDefinition,
+    behaviors: [
+      coreBehavior.blockObjects.breakingBlockObject,
+      coreBehavior.lists.indentListOnTab,
+      coreBehavior.lists.unindentListOnShiftTab,
+      coreBehavior.softReturn,
+    ],
+  }}
+>
+  {/* ... */}
+</EditorProvider>
+```
+
+### Behavior Events
+
+Behavior Events are separated into two categories:
+
+1. Native Behavior Events
+2. Synthetic Behavior Events
+
+Native events don't have a default action while synthetic events do. Synthetic events can also be triggered [imperatively](#imperative-events).
+
+#### Native Behavior Events
+
+- `copy`
+- `paste`
+- `key.down`
+- `key.up`
+
+#### Synthetic Behavior Events
+
+- `annotation.add`
+- `annotation.remove`
+- `annotation.toggle`
+- `blur`
+- `decorator.add`
+- `decorator.remove`
+- `decorator.toggle`
+- `delete.backward`
+- `delete.forward`
+- `focus`
+- `insert.block object`
+- `insert.inline object`
+- `insert.break`
+- `insert.sort break`
+- `insert.text`
+- `list item.toggle`
+- `style.toggle`
+
+### Imperative Events
+
+Setting up the editor using `EditorProvider` provides access to `useEditor()` which allows you to imperatively send synthetic events into the editor:
+
+```ts
+const editor = useEditor()
+```
+
+```tsx
+<button
+  onClick={() => {
+    props.editor.send({
+      type: 'decorator.toggle',
+      decorator: 'strong',
+    })
+    props.editor.send({type: 'focus'})
```

Comment:
mega nit: perhaps we don't need to include the `props.` here, since whether that editor instance is being passed to this component or not is irrelevant to this example